drivers/dai/intel/dmic/dmic.h

```diff
@@ -54,8 +45,6 @@
 #define TS_LOCAL_TSCTRL_HHTSE_BIT	BIT(7)
 #define TS_LOCAL_TSCTRL_ODTS_BIT	BIT(5)
 #define TS_LOCAL_TSCTRL_CDMAS(x)	SET_BITS(4, 0, x)
-#define TS_LOCAL_OFFS_FRM		GET_BITS(15, 12)
-#define TS_LOCAL_OFFS_CLK		GET_BITS(11, 0)
```

Comment:
did you mean to remove these?

Reply:
yep, these are  not being used ... it is even using wrongly the macro (missing one parameter)